### ISSUE TITLE
Good suggestion that I believe everyone would appreciate very much...

### ISSUE THREAD

**Issue by anonyxuser1** [mpv-player/mpv#6736]
Not having this little feature/addition has been causing problems for me using mpv so I am posting it here if that's alright, lol.

Rather than using settings such as `save-position-on-quit` which I believe is for the volume in the mpv.conf file. You should make a global setting such as `save-on-quit` for everything in the GUI that is changeable by the user. Also make this setting remember for all files and formats together, and not per file.

Hoping my suggestion can get put into the next version as I started liking the simplicity of mpv more than vlc lately, and now I really like the program, so much that some things need to be added to make it so much better, and that has also become important to me.

Your fan,
Anonymous

### mpv version and platform

If you're not using git master or the latest release, update.
Releases are listed here: https://github.com/mpv-player/mpv/releases

### Log file

Make a log file made with -v or --log-file=output.txt, paste it to
https://0x0.st/ or a similar site, and replace this text with a link to it.

Providing a log file is strongly encouraged. It is very helpful for reproducing
bugs or getting important technical context. Without a log it might not be
possible to analyze and fix certain bugs, and your bug report will be closed
or ignored.

### Sample files

Sample files needed to reproduce this issue can be uploaded to https://0x0.st/
or similar sites. (Only needed if the issue cannot be reproduced without it.)
Do not use garbage like "cloud storage", especially not Google Drive.

**Comment by lachs0r** [comment]
`save-position-on-quit` already does that. It saves most of the properties that can be modified for the list of files that were specified on the command line. The exact list is here: https://github.com/mpv-player/mpv/blob/master/player/configfiles.c#L209

If you think something should be added to this list, feel free to open a new issue.

**Comment by anonyxuser1** [comment]
Wait so it works globally for all files? And its called `save-position-on-quit` ? It should be `save-on-quit` because a lot of the options arent exactly positions. Well this is great news!

I tried it on the most common config option:
`script-opts=osc-visibility=save-position-on-quit` 
... but it had no effect on anything

Ahh, the above is not something thats on the list at the url you provided me.
Well, is there one for the above?

Me blabbing on:
Why would anyone want the settings to stick for a single file? I can understand the use for individual files for a setting like saving the time position of  how far you're into watching the video, but for settings, that just so useless to most, actually its a pain in the butt for most I would imagine. Some of these options need to be re-thought in a universally understandable method that makes it useful for most the world, and not just a few people. I can't imagine too many people wanting a setting where the gui is on screen and off-screen depending on which video they are watching, assuming they would ever watch the same video again when there is endless content on the Internet. If you want my suggestions with your program, I can be a great part of the team with my the experience I have a about asll the correct answers in knowing the worlds universal needs/wants, however success has a lot to do with getting your product high visibility, which can be a hard task. Or what would be the more/most used/needed out of what you might ask me if I was able to be apart of mpv team. I know how the brains work of the majority of the population and how to make things a success. Your program is already so popular so it wouldnt take long assuming you wanted to crank out some simple changes/additions to how mpv works. The results would make mpv #1 most useful and most used video player within the population of users that preferred a simple yet powerful video player (built on performance), and if its good enough they would be willing to give it a try and switch to mpv even when something like simplicity is not their biggest concern. I'd love to see that! Ok shutting up now, lol.

Thanks for your time.

Your fan, 
Anonymous
